guacamole/src/main/webapp/app/manage/controllers/manageUserController.js

```diff
@@ -130,16 +130,16 @@ angular.module('manage').controller('manageUserController', ['$scope', '$injecto
     permissionService.getPermissions(authenticationService.getCurrentUserID())
             .success(function permissionsReceived(permissions) {
                 
-         // Check if the user has UPDATE permission
-         $scope.hasUpdatePermission =
-               PermissionSet.hasSystemPermission(permissions, PermissionSet.SystemPermissionType.ADMINISTER)
-            || PermissionSet.hasUserPermission(permissions, PermissionSet.ObjectPermissionType.UPDATE, username);
-            
-         // Check if the user has DELETE permission
-         $scope.hasDeletePermission =
-               PermissionSet.hasSystemPermission(permissions, PermissionSet.SystemPermissionType.ADMINISTER)
-            || PermissionSet.hasUserPermission(permissions, PermissionSet.ObjectPermissionType.DELETE, username);
-    
+        // Check if the user is new or if the user has UPDATE permission
+        $scope.canSaveUser = !username || 
```

Comment:
When there is no identifier, and thus a user is creating a user on save, the user can save if and only if they have user creation permission... or is a sysadmin.